django/thunderstore/webhooks/models.py

```diff
@@ -60,25 +60,27 @@ def __str__(self):
     def get_for_package_release(cls, package):
         base_query = Q(Q(webhook_type=WebhookType.mod_release) & Q(is_active=True))
         community_query = Q()
-        community_exclusions = []
+        communities = []
         for listing in package.community_listings.all():
             if listing.review_status not in listing.community.valid_review_statuses:
-                community_exclusions.append(listing.community)
                 continue
             categories = listing.categories.all()
             query = (
                 ~Q(exclude_categories__in=categories)
                 & Q(Q(require_categories=None) | Q(require_categories__in=categories))
                 & Q(community_site__community=listing.community)
             )
+            communities.append(listing.community)
```

Comment:
I find the placement of this row a bit odd (between the rows for building the query). I would place it directly after the `if` block or at the bottom of the current block.

Reply:
The grouping I had in mind here was that the filter applied a line above this was also related to the community, but fair enough